src/Components/TodoEdit.tsx

```diff
@@ -0,0 +1,35 @@
+import { SetStateAction, useState } from 'react';
+import { Todo } from '../types/Todo';
+
+export const TodoEditForm = ({ todo, onEdit }
+: { todo: Todo; onEdit: (editedTodo: Todo) => void }) => {
+  const [title, setTitle] = useState(todo.title);
+
+  const handleTitleChange
+  = (event: { target: { value: SetStateAction<string>; }; }) => {
```

Comment:
The type of 'event.target.value' is not 'SetStateAction<string>'. It should be 'string'. SetStateAction is used with the setState function, not for event handling.